lib/systems/include/PedalsSystem.h

```diff
@@ -28,68 +39,55 @@ struct PedalsParams
     float mechanical_activation_percentage;
 };
 
+
+
 class PedalsSystem
 {
 public:
+    struct PedalData_s {
```

Comment:
Why is this defined as a nested struct within PedalsSystem? Why not use the globally-defined pedals system data struct